src/ErrorHandling.h

```diff
@@ -170,6 +170,7 @@ typedef enum
     SUBCODE_CONFIGURATION_FAIL_MP_NICDATA1,
     SUBCODE_FAIL_MP_NICDATA_INIT1,
     SUBCODE_FAIL_INVALID_BASE_TRACK_MOTION_TYPE,
+    SUBCODE_DEBUG_INIT_FAIL_MP_NICDATA_SPECIFIC,
```

Comment:
what's *specific* about it?

Reply:
A previous commit had `SUBCODE_DEBUG_INIT_FAIL_MP_NICDATA_ALL` as a code, but now that has been removed so I could get rid of the `_SPECIFIC`